code/controllers/CMSPageHistoryController.php

```diff
@@ -358,7 +358,7 @@ public function ShowVersionForm($versionID = null) {
 	/**
 	 * @param int $versionID
 	 * @param int $otherVersionID
-	 * @return Form
+	 * @return mixed Form|boolean|SS_HTTPResponse
```

Comment:
> let's not over think this. This is how the method has been for years

Aceepted, the syntax is still wrong, though - would you mind doing one of either `mixed` or the types it can return?